leaflet-maplibre-gl.js

```diff
@@ -7,7 +7,8 @@
         module.exports = factory(require('leaflet'), require('maplibre-gl'));
     } else {
         // Browser globals (root is window)
-        root.returnExports = factory(window.L, window.maplibregl);
+        root = typeof globalThis !== 'undefined' ? globalThis : root || self;
+        root.returnExports = factory(root.L, root.maplibregl);
     }
 }(this, function (L, maplibregl) {
```

Comment:
How about this ?
```suggestion
}(typeof globalThis !== 'undefined' ? globalThis : root || self, function (L, maplibregl) {
```

Reply:
I wasn't sure about which way and followed what Rollup does, but this looks like the nicer solution indeed.

Added as an additional commit, let me know if you want it amended instead or feel free to squash on merge.